dateparser/custom_lang_detect/lang_detect.py

```diff
@@ -0,0 +1,19 @@
+import langdetect
+from dateparser.conf import apply_settings
+
+langdetect.DetectorFactory.seed = 0
+
+@apply_settings
+def detect_languages(text, settings=None):
+    language_codes = ["en"] 
+
+    try:
+        parser_data = str(langdetect.detect_langs(text)[0]).split(":")
+        confidence_score = float(parser_data[1])
```

Comment:
Do we need to convert it to string? I think it should be possible to get probabilities directly.

Also, don't we want to get all language detections?

Reply:
I am fixing